multinet/api/tests/utils.py

```diff
@@ -1,7 +1,35 @@
 from typing import Dict, List
 
+from django.contrib.auth.models import User
 from rest_framework.test import APIClient
 
+from multinet.api.models import Workspace
+from multinet.api.tests.factories import UserFactory
+from multinet.api.utils.workspace_permissions import WorkspacePermission
+
+
+def create_users_with_permissions(user_factory: UserFactory, workspace: Workspace):
+    for permission in [
+        WorkspacePermission.reader,
+        WorkspacePermission.writer,
+        WorkspacePermission.maintainer,
+    ]:
+        user_list: List[User] = [user_factory() for _ in range(3)]
+        for user in user_list:
+            workspace.set_user_permission(user, permission)
+
+
+def get_field_value(data_file, s3ff_client):
+
+    field_value = ''
```

Comment:
You don't need this, since due to python's scope rules, `field_value` will still be in scope after the `with .. as` block.
```suggestion
```